app/controllers/items_controller.rb

```diff
@@ -5,7 +5,8 @@ def index
     if params[:query].present?
       @items = Item.search_index(params[:query])
     else
-      @items = Item.all.order(id: :desc)
+      # Changed to :asc for the Thursday demo
```

Comment:
Otherwise, the spaghetti meal is at the very bottom. This can be changed again afterwards